### DISCUSSION TITLE
🧩 FEATURE REQUEST: Allow people to add their own descriptions to custom shortcuts. [A11y][UX]

### DISCUSSION THREAD

**Discussion by InHuOfficial** [dailydotdev/daily#1013] [Feature Requests]
## Feature Request
Give the option to add a short description to custom shortcuts.

### Description
For people using assistive technology the new shortcuts use the site URL as the accessible text (the `alt` attribute on the image within the anchor).

This is not an ideal solution as if the bookmark contains a load of identification characters as part of a key or they have 2 bookmarks for the same site it means they cannot differentiate easily.

Instead being able to have some user defined short text to describe the bookmark would be better.

### Solution
Make the modal wider for adding shortcuts and add a second column "bookmark description (optional)".

This should then be used as the `alt` text on the image if it is set (if not fall back to the URL as alt text as it is currently).

Additionally this would be great if this is set as as the `title` attribute on the anchor so you can hover over icons for the same site but different endpoints, in order to differentiate them.

### Alternate Solutions
Alternatively allowing people to upload their own icons may be an option to solve the differentiation part for 2 links to the samebase domain, it doesn't fix the accessibility issue though.

**Comment by rebelchris** [comment]
Great suggestion indeed
I think we should indeed look into this, as multiple people requested this feature. 🙌
